Automattic/wp-calypso
# [Odie] Refine UI with animations and better UX

Related to: https://github.com/Automattic/ai-services/issues/555

## Proposed Changes

- Add new animations
- Remove pointer events in the white gradient
- Scroll down when typing
- Remove svgs and convert them to React Components
- Adjusted some sizes (there were double scrolls)
- Adds default config to the Help Center (@alshakero)
- Adds sources and simulate typing
- Clear chats option enabled

## Testing Instructions

Need to apply the wpcom patch 126768 or wait for it to be deployed once finished/reviewed

There should not be any breaking change (hopefully)
1. Play with the Help Center without the flag added, check for no errors at all while playing with it
2. Append the `?flags=wapuu` in the URL, then click the Help Center and Click "Still need help?"
3. Use the feedback button and assert that everything keeps workign
4. Scroll halfway to the bottom of the messages, and start typing, it should scroll to the bottom.
5. Talk a bit with Wapuu
6. Assert that the Start a new chat button works (three dots menu at the top of the Help Center conversation)

https://github.com/Automattic/wp-calypso/assets/5689927/5e61a146-e10e-4354-bc7d-00712e56d39c


## Pre-merge Checklist
